src/odinapi/views/baseview.py

```diff
@@ -169,3 +180,15 @@ def _swagger_spec(self, version):
             return
         if version in self.VERSION_TO_SWAGGERSPEC:
             return getattr(self, self.VERSION_TO_SWAGGERSPEC[version])(version)
+
+
+def default(obj):
```

Comment:
What about `str`?

Reply:
Oh, you were too late. This is only how to handle things that are not valid json I believe, and strings are valid.